src/table/__tests__/resizable-columns.test.tsx

```diff
@@ -323,14 +323,14 @@ describe('resize with keyboard', () => {
 
     await waitFor(() => {
       expect(onChange).toHaveBeenCalledTimes(1);
-      expect(onChange).toHaveBeenCalledWith({ widths: [150 - 10, 300] });
+      expect(onChange).toHaveBeenCalledWith({ widths: [150 - 10, 300], resizedColumn: { id: 'id', width: 150 - 10 } });
```

Comment:
nit, non blocking: no need to change, since it was already like this, but these "10" pixels could probably be a const with a descriptive name "KEYBOARD_RESIZE_STEP".